### ISSUE TITLE
lvm OSD provisioning fails if docker is not installed (stable-3.2)

### ISSUE THREAD

**Issue by discostur** [ceph/ceph-ansible#4498]
Hi,

if you try to setup a new ceph cluster with stable-3.2 it fails in the lvm / osd setup:

```
TASK [ceph-config : run 'ceph-volume lvm batch --report' to see how many osds are to be created] *******************************************************************************************************************************************************************************
Monday 30 September 2019  17:15:04 +0200 (0:00:00.157)       0:02:43.188 ****** 
fatal: [ceph-osd02-dev.intern]: FAILED! => {"changed": false, "cmd": "docker run --rm --privileged --net=host --ipc=host --ulimit nofile=1024:4096 -v /run/lock/lvm:/run/lock/lvm:z -v /var/run/udev/:/var/run/udev/:z -v /dev:/dev -v /etc/ceph:/etc/ceph:z -v /run/lvm/lvmetad.socket:/run/lvm/lvmetad.socket -v /var/lib/ceph/:/var/lib/ceph/:z -v /var/log/ceph/:/var/log/ceph/:z --entrypoint=ceph-volume docker.io/ceph/daemon:latest-mimic --cluster ceph lvm batch --bluestore --yes --prepare /dev/sdb /dev/sdc /dev/sdd /dev/sde --report --format=json", "msg": "[Errno 2] Datei oder Verzeichnis nicht gefunden", "rc": 2}
``` 

Containerized deployment is disabled:

```
containerized_deployment: False
```

---
ansible 2.6.19
centos-7
ceph-ansible stable-3.2

**Comment by dsavineau** [comment]
Please share your full ansible logs, inventory and variables defined in [host|group]_vars/*.yml

**Comment by dsavineau** [comment]
Ok I think I found the issue no need to share the logs.

**Comment by discostur** [comment]
@dsavineau manually applied your changes to my ansible playbook - seems to work! thanks ;)

**Comment by dsavineau** [comment]
@discostur FYI commit merged into stable-3.2 branch. This will be part of the next v3.2.29 release.
